utils/metrics/metrics/src/main/java/com/ea/orbit/metrics/MetricsComponent.java

```diff
@@ -36,10 +36,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.inject.Singleton;
+
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.List;
 
+@Singleton
```

Comment:
Yes, that's normal. Only actual singleton components will be started by the framework on it's own, otherwise it doesn't know what the lifetime should be.